docs/src/main/asciidoc/property-source-config.adoc

```diff
@@ -400,6 +400,50 @@ will result in three properties being generated:
 
  - `config-map-three.greetings.message` equal to `Say Hello from three`.
 
+By default, besides reading the config map that is specified in the `sources` configuration, Spring will also try to read
+all properties from "profile aware" sources. The easiest way to explain this is via an example. Let's suppose your application
+enables a profile called "dev" and you have a configuration like the one below:
+
+====
+[source,yaml]
+----
+spring:
+  application:
+    name: spring-k8s
+  cloud:
+    kubernetes:
+      config:
+        namespace: default-namespace
+        sources:
+          - name: config-map-one
+----
+====
+
+Besides reading the `config-map-one`, Spring will also try to read `config-map-one-dev`; in this particular order. Each active profile
+generates such a profile aware config map.
+
+Though your application should not be impacted by such a config map, it can be disabled if needed:
+
+====
+[source,yaml]
+----
+spring:
+  application:
+    name: spring-k8s
+  cloud:
+    kubernetes:
+      config:
+        useProfileNameAsSuffix: false
```

Comment:
This property name is confusing to me, thats why I didn't understand the purpose of this PR to begin with ;)

Can we change it to something like `includeProfileSpecificSources` or `includeProdileSpecificConfiguration`?

Reply:
darn! `includeProfileSpecificSources` is a very nice name. I will adjust the PR and ping you when ready